### PR TITLE
Import PEP 440

### PR DESCRIPTION
**Pull request by jeanas**
Relates to #1093.

### CODE REVIEW

**Review by sinoroc — COMMENTED** [review]
Just having this page here and linked correctly is such an improvement!

I assume it is mostly copy-paste from the PEPs, is that right? Great job in any case!

File: source/specifications/version-specifiers.rst [hunk 1]
```diff
@@ -47,17 +19,6 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
...
-Projects include Python libraries, frameworks, scripts, plugins,
-applications, collections of data or other resources, and various
-combinations thereof. Public Python projects are typically registered on
-the `Python Package Index <https://pypi.org/>`__.
-
-"Releases" are uniquely identified snapshots of a project.
-
-"Distributions" are the packaged files which are used to publish
-and distribute a release.
-
```
[the remark this answers is not in the document]
Comment (sinoroc) [reply source/specifications/version-specifiers.rst]: Typically how I handle these cases, is that I keep things as-is and try to get them merged soon, but at the same time I open an issue ticket as a reminder that we might want to do a second pass later on and fix some things.

**Review by sinoroc — APPROVED** [review]

**Review by pradyunsg — APPROVED** [review]
I haven't double-checked that the contents of the newly-created page actually matches PEP 440 in a 1:1 manner, and am trusting that @jeanas hasn't sneaked in a spec change here. :)

**Comment by sinoroc** [comment]
@jeanas Thank you so much! So many times I have wanted to send someone a link to the specification for version specifiers but felt bad about sending the link to the PEP(s).

**Comment by hugovk** [comment]
Please see PR https://github.com/pypa/packaging.python.org/pull/1373 for a follow-up to replace "PEP" with "specification".

**Pull request merged**